plasmapy/plasma/grids.py

```diff
@@ -513,6 +539,14 @@ def _load_grid(
         self.ds = xr.Dataset()
 
         self.ds.attrs["axis_units"] = [pts0.unit, pts1.unit, pts2.unit]
+
+        # Store the conversion factors for each axis to SI
+        self.si_factors = [
+            (1 * pts0.unit).si.value,
+            (1 * pts1.unit).si.value,
+            (1 * pts2.unit).si.value,
```

Comment:
```suggestion
            pts0.unit.si.scale,
            pts1.unit.si.scale,
            pts2.unit.si.scale,
```

Since we know converting to units and stripping units are painfully slow, we should always be looking to avoid/minimize these actions.  The current implementation does both of these slow processes, my suggestion does neither.

![image](https://user-images.githubusercontent.com/29869348/127893191-6990c62c-c573-4e5e-bd45-2a62caef86e8.png)

Reply:
Oh nice! I didn't know about this scale attribute.